### PR TITLE
Add an extra output package for ibis-core

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR aims to create a new output just for ibis-framework-core that will allow other external backends to use ibis with a more compact dependency tree. it will keep the current package for ibis-framework with no changes.

PS: let me know if it is ok for this feedstock this changes, if it is not ok here I can create a new feedstock just for the core.

### CODE REVIEW

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

**Comment by xmnlab** [comment]
@jreback @cpcloud @datapythonista if you have time, could you review this PR pls?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,69 +1,106 @@
...
+        - ibis.backends.postgres
+        - ibis.backends.pyspark
+        - ibis.backends.sqlite
+
+  - name: ibis-framework-core
+    version: {{ version }}
+
+    build:
+      script: python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+      skip: true  # [py<37]
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: This can probably be `noarch: python` with a minimum python version of 3.7

Comment (xmnlab) [reply recipe/meta.yaml]: thanks for the feedback. I will update the recipe! thanks!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,69 +1,107 @@
...
-    - pytz
-    - regex
-    - requests
-    - shapely
-    - setuptools
-    - sqlalchemy >=1.1
-    - thrift >=0.11
-    - thriftpy2
-    - toolz
+  - name: ibis-framework
```
Comment (cpcloud) [inline recipe/meta.yaml:48]: Can this depend on `ibis-framework-core` instead of duplicating the dependencies?

Comment (xmnlab) [reply recipe/meta.yaml:48]: mm I guess it makes totally sense, let me try to see if it will work :) thanks for the feedback!

Comment (xmnlab) [reply recipe/meta.yaml:48]: it seems that worked :) thanks @cpcloud !

**Comment by xmnlab** [comment]
hey all, any more thoughts about this PR?

**Comment by xmnlab** [comment]
@jreback @cpcloud @datapythonista @mariusvniekerk any more thoughts about this PR? thanks a lot!

**Comment by xmnlab** [comment]
Hi all, any more thoughts about this PR? this is very critical to have any external backend installable via conda

**Review by cpcloud — APPROVED** [review]

**Comment by xmnlab** [comment]
thanks @cpcloud !

**Comment by jreback** [comment]
thanks @xmnlab

**Pull request merged**